grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceTest.java

```diff
@@ -119,4 +124,17 @@ public void missingMethod() throws Exception {
                            .set(HttpHeaderNames.CONTENT_LENGTH, "0"),
                 HttpData.of(new byte[] {})));
     }
+
+    @Test
+    public void pathMapping() throws Exception {
+        assertThat(grpcService.pathMappings())
+            .isEqualTo(
+                TestServiceGrpc.getServiceDescriptor()
+                    .getMethods()
+                    .stream()
+                    .map(MethodDescriptor::getFullMethodName)
+                    .sorted(String::compareTo)
```

Comment:
Ditto: `sorted()` should be enough. Or we won't need it at all if we switch to `Set<PathMapping>`